test/addons/rbd-mirror/start

```diff
@@ -33,7 +52,7 @@ def fetch_secret_info(cluster):
 
     print(f"Getting mirroring info site name for cluster '{cluster}'")
     info["name"] = drenv.wait_for(
-        f"cephblockpools.ceph.rook.io/{POOL_NAME}",
+        f"cephblockpools.ceph.rook.io/{POOL_NAMES[0]}",
```

Comment:
With the simplified configuration I suggested, this will become:

    CONFIG[0]["pool"]